### DISCUSSION TITLE
Do we need to add one by default when the Table has no sorting items?

### DISCUSSION THREAD

**Discussion by SmRiley** [ant-design-blazor/ant-design-blazor#3146] [Ideas]
By default, if skip/take is used (required for paging), it will cause ef to issue this warning. Although it does not affect the use, can we add a default sort item to suppress it?
![image](https://user-images.githubusercontent.com/45205313/221900944-da24b284-efff-4302-b951-9b52259b37d3.png)

**Comment by ElderJames** [comment]
Hello @SmRiley , which field do you think should be sorted by default? And how to specific it by user?
